### ISSUE TITLE
ChatGLM Endpoint URL is invalid.

### ISSUE THREAD

**Issue by Flagami** [langgenius/dify#1854]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to file this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).

### Dify version

0.3.21

### Cloud or Self Hosted

Self Hosted (Docker)

### Steps to reproduce

![image](https://github.com/langgenius/dify/assets/125533116/ab3a9fbe-37ad-4483-b613-af6b52f8adab)
![image](https://github.com/langgenius/dify/assets/125533116/1f795c52-8569-475b-acfe-c4003d38ebe0)

### ✔️ Expected Behavior

add the api successfully

### ❌ Actual Behavior

ChatGLM Endpoint URL is invalid.

**Comment by Flagami** [comment]
sorry the correct version is 0.3.34, please be noted!

**Comment by crazywoola** [comment]
Seems there is no input field.

**Comment by Flagami** [comment]
Nop, I just hidden it by cutting it out of the screenshot imageSent from Flag‘s Mobile PhoneOn 30 Dec 2023, at 21:46, crazywoola ***@***.***> wrote:﻿
Seems there is no input field.

**Comment by angelala00** [comment]
I have the same problem and I noticed that dify request uri '/v1/models' when it verify the custom api domain.
So I tried deploy chatglm with fastchat (provides OpenAI-compatible APIs), then the api successfully added, and I create a new app. 
When I use this app to chat with model, I got another error : "The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application."
